pycbc/events/significance.py

```diff
@@ -187,6 +187,14 @@ def get_n_louder(back_stat, fore_stat, dec_facs,
     Wrapper to find the correct n_louder calculation method using standard
     inputs
     """
+    # This nan to inf conversion is a safety shield; there shouldn't be any
+    # nan statistic values, but if there are, they should be considered as
+    # quiet as possible. nans are considered greater than any floats in an
+    # argsort
+    nanmask = np.isnan(back_stat)
+    bstat = copy.deepcopy(back_stat)
+    back_stat[nanmask] = -np.inf
+
```

Comment:
Can you add some logging for the case where there are nans? I like having this shield in place, but we need to know if there are nans making it in somehow.

Reply:
should this be `bstat[nanmask] = -np.inf` ? 
I'm not really seeing why the deepcopy is made either

Reply:
That was a typo on line 195 rather than anything clever. The deepcopy is made so that back_stat is not modified in-place